README.md

```diff
@@ -1,16 +1,14 @@
 # CPAN.org content repository
 
 This repository is for maintaining the non-source-code, non-archive
-content on cpan.org.  Discussion related to this should be on the
+content on [cpan.org][cpan].  Discussion related to this should be on the
 [cpan-workers mailing list][cpan-workers].
```

Comment:
I'd prefer to either have the links be inline `[cpan-workers mailing list](https://lists.perl.org/list/cpan-workers.html)` or nearby to use like they were.  When they're down at the bottom, it's easy to get confused or out of date.

Reply:
I'll inline then